app/controllers/passengers_controller.rb

```diff
@@ -33,7 +32,7 @@ def create
   end
 
   def update
-    if @passenger.update(passenger_params)
+    if @passenger.update_attributes(passenger_params)
```

Comment:
Dissagree?

http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attributes

Api docs for `#update_attributes` says "alias for `#update`"